docs/known_issues.md

```diff
@@ -28,7 +28,7 @@ class Product
     {
         $this->imageFile = $imageFile;
 
-        // Only change the updated af if the file is really uploaded to avoid database updates.
+        // Only change the updated at if the file is really uploaded to avoid database updates.
```

Comment:
Nice spotted.
Anyway, it should be "updatedAt" instead. That's the name of the property.